recipes/miniaudio/all/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.4)
+project(miniaudio C)
+
+include(GNUInstallDirs)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(KEEP_RPATHS)
```

to create relocatable shared lib on macOS